packages/toolpad-app/src/config.ts

```diff
@@ -30,7 +30,9 @@ export type BuildEnvVars = Record<
   // The current Toolpad version
   | 'TOOLPAD_VERSION'
   // The current Toolpad build number
-  | 'TOOLPAD_BUILD',
+  | 'TOOLPAD_BUILD'
+  | 'IS_SENTRY_ENABLED'
+  | 'SENTRY_DSN',
```

Comment:
sure this is build time only? i.e. we bake it in the image? self hosters can't add their own sentry?

Reply:
> not defining the TOOLPAD_SENTRY_DSN variable means disabling sentry. no need to add more redundant variables. also, I'd keep the convention of prefixing Toolpad variables with TOOLPAD_

Makes sense, I was thinking of this env variable as more like an on/off toggle but you're right, it's redundant, better to just use SENTRY_DSN.